frontend/class-twitter.php

```diff
@@ -161,6 +161,9 @@ protected function description() {
 		elseif ( WPSEO_Frontend::get_instance()->is_posts_page() ) {
```

Comment:
This function probably will be more readable if the statements are added in a switch statement, instead of a multiple if statement